common/src/sports-info.ts

```diff
@@ -0,0 +1,10 @@
+export interface Fixture {
```

Comment:
Please rename to sth like `SportsGame` or `Game`
<img width="642" alt="Screenshot 2024-12-17 at 1 59 23 PM" src="https://github.com/user-attachments/assets/34c00355-a468-43ad-aac3-5611de3d6c09" />